infra/modules/emails/main.tf

```diff
@@ -0,0 +1,102 @@
+variable "email_domains" {
```

Comment:
ideally, we don't want these emails to be checked in. if this was closed source it wouldn't be too bad, but since we are open source bots will look through repos and potentially end up spamming these emails. better to have them as variables. 

had to do the same here: https://github.com/JiscSD/octopus/pull/390/commits/23ede00c68f3a770204523e345bc5dfc75764892

terraform will ask you for them at the plan/apply stage or you can directly include them in int/prod.tfvars but don't check them in etc.